tests/Queue/QueueIronJobTest.php

```diff
@@ -21,7 +21,7 @@ public function testFireProperlyCallsTheJobHandler()
     public function testDeleteRemovesTheJobFromIron()
     {
         $job = $this->getJob();
-        $job->getIron()->shouldReceive('deleteMessage')->once()->with('default', 1);
+        $job->getIron()->shouldReceive('deleteMessage')->once()->with('default', 1, null);
```

Comment:
it's important that this test passes without changing it to prove the changes are non-breaking